blas/testblas/benchautogen/autogen_bench_level1double.go

```diff
@@ -219,8 +216,7 @@ func level1(benchPath string, pkgname string) error {
 	level1Filepath := filepath.Join(benchPath, "level1doubleBench_auto_test.go")
 	f, err := os.Create(level1Filepath)
 	if err != nil {
-		fmt.Println(err)
-		os.Exit(1)
+		return err
```

Comment:
that's a fine change but it shouldn't probably be part of that CL which deals with license stuff, should it?

Reply:
The code generator here needed to be fixed; it wouldn't actually run without changes. I think this is a minor change in the context.

Reply:
ok.